src/Moat.hs

```diff
@@ -860,16 +864,17 @@ mkTypeAlias typName instTys = \case
 -- | Make a void type (empty enum)
 mkVoid ::
   () =>
+  Options ->
```

Comment:
Feels like a good use of `ReaderT Options` - maybe put that in with `MoatM`?

Reply:
ah lol i see that you already have this as a refactoring potential!

Reply:
It is definitely getting tedious to pass these around over and over.